programs/survival/fakeGui.sc

```diff
@@ -0,0 +1,461 @@
+        'menu' -> 'Menu',
+        'who_menu' -> '\'s menu',
+        'left_click' -> 'Left click',
+        'right_click' -> 'Right click',
+        'backpack' -> 'Backpack',
+        'switch_off-hand_items' -> 'Switch off-hand items',
+        'jump' -> 'Jump',
+        'stop' -> 'Stop the Fake player',
+        'remove' -> 'Remove the Fake player',
+        'not_fake' -> 'This player is not a Fake player.',
+        'Toolbar' -> 'Toolbar',
+        'equipment_bar' -> 'Equipment bar',
+        'attack_mode' -> 'Attack mode',
+        'use_mode' -> 'Use mode',
+        'set_interval' -> 'Set interval',
+        'add' -> 'Add',
+        'reduce' -> 'Reduce',
+    },
+    'zh_cn' -> {
+        'usage' -> '\n如何使用:\n    右键点击假人\n或是:\n    执行指令 /fakegui <player>',
+        'click_once' -> '点击一次',
+        'timing_click' -> '间隔点击',
+        'keep_clicking' -> '持续点击',
+        'menu' -> '菜单',
+        'who_menu' -> ' 的菜单',
+        'left_click' -> '左键点击',
+        'right_click' -> '右键点击',
+        'backpack' -> '背包',
+        'switch_off-hand_items' -> '副手切换',
+        'jump' -> '跳',
+        'stop' -> '停止假人动作',
+        'remove' -> '移除假人',
+        'not_fake' -> '他/它 不是假人',
+        'Toolbar' -> '工具栏',
+        'equipment_bar' -> '装备栏',
+        'attack_mode' -> '攻击模式',
+        'use_mode' -> '使用模式',
+        'set_interval' -> '设定间隔',
+        'add' -> '添加',
+        'reduce' -> '减少',
+    },
+};
+
+global_slotmap = [[-1,7],[-2,0],[-3,1],[-4,2],[-5,3]];
+global_fakeplayers_selected_slot = m();
+global_fakeplayersscreen = m();
+global_counts = m();
+global_models(player) -> l(
+    nbt('{ name: \'once\', title: \'§f§'+i18n(player, 'click_once')+'\', item: \'minecraft:minecart\'}'),
+    nbt('{ name: \'continuous\', title: \'§f§L'+i18n(player, 'keep_clicking')+'\', item: \'minecraft:apple\'}'),
+    nbt('{ name: \'interval\', title: \'§f§L'+i18n(player, 'timing_click')+'\', item: \'minecraft:repeater\'}'),
+);
+
+__page(type_, creativeplayer, fakeplayer) -> (
+    if (
+        // 菜單
+        type_ == 'menu', (
+            screen = create_screen(creativeplayer, 'generic_9x3', fakeplayer~'display_name'+i18n(creativeplayer, 'who_menu'), _(screen, player, action, data, outer(fakeplayer)) -> (
+                slot = data:'slot';
+                if(
+                    action == 'close', (
+                        global_fakeplayersscreen:fakeplayer = null;
+                        drop_item(screen, -1);
+                    ),
+                    action == 'pickup', 
+                    if (
+                        slot > 9*3-1, null,
+                        (
+                            newPage = command = null;
+
+                            if (
+                                slot == 16, command = 'kill',
+                                slot == 15, command = 'stop',
+                                slot == 14, command = 'jump',
+                                slot == 13, command = 'swapHands'
+                            );
```

Comment:
Instead of checking each explicitly you could use a map instead
```
global_slot_commands = {
  16 -> 'kill',
  15 -> 'stop',
  14 -> 'jump',
  13 -> 'swapHands'
}
```
```suggestion
                          command = global_slot_commands:slot
```
You could even use a more complex map with pages
```
global_slot_commands = {
  'bag' -> {
    16 -> 'kill',
    ...
  },
  'use' -> {
    ...
  },
  'attack'-{
    ...
  }
  
}
...
command = global_slot_commands:current_page:slot
```